test/test_helper.rb

```diff
@@ -16,8 +16,11 @@
 # <http://www.gnu.org/licenses/agpl.html>
 #
 require 'minitest'
-require 'simplecov'
-SimpleCov.start if (!ENV.has_key?('COV') && !ENV.has_key?('COVERAGE')) || (ENV['COV'] != 'false' && ENV['COVERAGE'] != 'false')
+
+if Rake.application.top_level_tasks.include?('test') && (!ENV.key?('COV') && !ENV.key?('COVERAGE')) || (ENV['COV'] != 'false' && ENV['COVERAGE'] != 'false')
```

Comment:
If we want to keep it ON by default then I suggest the following which is more natural/easy to read 
```suggestion
if Rake.application.top_level_tasks.include?('test') && [ENV['COVERAGE'], ENV['COV']].exclude? 'false'
```